File: app/src/main/java/com/soya/launcher/ui/fragment/HomeGuideGroupGradientFragment.java
package com.soya.launcher.ui.fragment;

import android.os.Bundle;

public class HomeGuideGroupGradientFragment extends AbsGroupGradientFragment{
    public static HomeGuideGroupGradientFragment newInstance() {

        Bundle args = new Bundle();

        HomeGuideGroupGradientFragment fragment = new HomeGuideGroupGradientFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    protected boolean isGuide() {
        return false;
    }
}

File: app/src/main/java/com/soya/launcher/ui/fragment/HomeGuideGroupGradientFragment.kt
package com.soya.launcher.ui.fragment

import android.os.Bundle
import com.shudong.lib_base.base.BaseViewModel
import com.soya.launcher.databinding.FragmentGuideGroupGradientBinding

class HomeGuideGroupGradientFragment : AbsGroupGradientFragment<FragmentGuideGroupGradientBinding,BaseViewModel>() {
    override val isGuide: Boolean
        get() = false

    companion object {
        fun newInstance(): HomeGuideGroupGradientFragment {
            val args = Bundle()

            val fragment = HomeGuideGroupGradientFragment()
            fragment.arguments = args
            return fragment
        }
    }
}
